app/views/buyers/applications/new.html.slim

```diff
@@ -0,0 +1,8 @@
+- if @account
+  = render '/buyers/accounts/menu'
```

Comment:
Hmmm this is strange :) why `@account` is the condition? Not explicit enough, especially when you do not use it anymore in the rest of the file

Reply:
If there's an account, print the account's menu.